klaytnetl/service/klaytn_contract_service.py

```diff
@@ -86,6 +86,20 @@ def is_erc721_contract(self, function_sighashes):
             and c.implements("approve(address,uint256)")
         )
 
+    # https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1155.md
+    # https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC1155/ERC1155.sol
+    # Found cases where balanceOf was not implemented
```

Comment:
What is the rule for this? AFAIK, the all functions ***must*** be implemented based on specification. Should `balanceOf` and `balanceOfBatch` be implemented both? How about `safeTransferFrom` and `safeBatchTransferFrom`?

Reply:
Okay! I will fix this case.